bin/build-dao.sh

```diff
@@ -8,34 +8,27 @@
 # entities (Civi's and your extension), and finally it copies your extension's
 # DAOs to your extension CRM/NAMESPACE/DAO directory.
 
-CIVIROOT="$1"
-EXTROOT="$2"
-NAMESPACE="$3"
-EXTSRCDIR="$EXTROOT/CRM/$NAMESPACE"
-EXTDAODIR="$EXTSRCDIR/DAO"
-XMLBUILD="$EXTROOT/build/xml/schema"
-
-function validateOptions() {
-  if [ -z "$CIVIROOT" -o ! -d "$CIVIROOT" ]; then
-    echo "ERROR: invalid civicrm-dir: [$CIVIROOT]"
-    printUsage
+function validatePaths() {
+  if [ -z "$CIVIROOT" ]; then
```

Comment:
Any reason you decided to remove the check if it's a directory?

Reply:
Based on the tests I did, the `cv path -d '[civicrm.root]` command either returns a valid path, when it can find one or empty when it can't. Based on that, it's ok to assume that if it's not empty than it exists and it's a directory. Basically, `cv` is the one doing that validation now